proto/zitadel/webkey/v2beta/webkey_service.proto

```diff
@@ -1,25 +1,23 @@
 syntax = "proto3";
 
-package zitadel.resources.webkey.v3alpha;
+package zitadel.webkey.v2beta;
 
 import "google/api/annotations.proto";
 import "google/api/field_behavior.proto";
+import "google/protobuf/timestamp.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 import "validate/validate.proto";
-
 import "zitadel/protoc_gen_zitadel/v2/options.proto";
+import "zitadel/webkey/v2beta/key.proto";
+import "zitadel/object/v2beta/object.proto";
```

Comment:
Do we still need the object.proto import?

Reply:
good catch. now cleaned up.